### DISCUSSION TITLE
[의견] 캘린더 일정 알림 스캐줄러 관련 서버 분리

### DISCUSSION THREAD

**Discussion by lcomment** [wooda-ege/lovebird-server#24] [Ideas]
@YongsHub 

현재 Prod 서버가 t2.small 스펙의 작은 ec2 인스턴스로 돌아가고 있는데, 내부적으로 캘린더 일정 알림에 대한 스케줄러가 계속 돌아가는게 조금 걱정이 됩니다. 이번 리펙토링 과정에서 Scheduler 관련 기능을 모듈로 분리하여 프리티어의 작은 서버로 따로 띄우는건 어떨지 의견드립니다.

**Comment by YongsHub** [comment]
좋은 것 같습니다. 아니면 Prod 서버 스펙 향상을 시키는것도 좋을 것 같은데 그 전에 Dummy를 처리하는 예시를 이용해서 Scheduler를 돌려보고 cpu나 memory 점유율 등 타당한 이유를 찾으면 더 합리적인 선택이 될 것 같다는 개인적인 의견을 말씀드려봅니다.

**Comment by lcomment** [comment]
scale-up은 비용상 조금 지양하고 있긴 해요 ㅜ 더미를 처리하는 테스트를 통한 도입은 찬성입니다.
스케줄러 서버를 미리 구현해 놓고 이슈가 있을 시에 배포하는 것도 괜찮아보이네요 생각해보니 !
멍 때리다가 생각나서 올린 discussion이었습니다
